keras_applications/densenet.py

```diff
@@ -179,7 +179,7 @@ def DenseNet(blocks,
                          'or the path to the weights file to be loaded.')
 
     if weights == 'imagenet' and include_top and classes != 1000:
-        raise ValueError('If using `weights` as imagenet with `include_top`'
+        raise ValueError('If using `weights` as \'imagenet\' with `include_top`'
```

Comment:
I mean \`'imagenet'\` not \\'imagenet\\'. Sorry for the confusion.

Reply:
Please see an [example](https://github.com/keras-team/keras-applications/blob/master/keras_applications/inception_resnet_v2.py#L121).

Reply:
Oh, I misunderstood. Got it now.